### PR TITLE
rl: fixed & tuned idle task rewards

### PR DESCRIPTION
**Pull request by AugustDG**
Small fixes & adjustments to training for the idle task:
- adjusted base learning rate
- added flag to avoid resetting when playing a checkpoint
- simplified paths within NewtonAgent construction
- moved Newton's parts within USD to a new "trans" parent 
- cached influxdb bucket queries

### CODE REVIEW

File: core/envs/newton_terrain_env.py [hunk 1]
```diff
@@ -1,8 +1,6 @@
 from typing import Optional
 
 import torch as th
```
Comment (Horisofine) [inline core/envs/newton_terrain_env.py:3]: Th?? Foul play!

**Review by Horisofine — APPROVED** [review]
Approve the terrain refactor! LGTM.
